docs/configuration/configuration.md

```diff
@@ -47,53 +47,44 @@ The other placeholders are specified separately.
 
 A valid example file can be found [here](/config/testdata/conf.good.yml).
 
-The global configuration specifies parameters that are valid in all other configuration
+### `<global>`
+
+The `global` section contains the global Prometheus server
+configuration. The global configuration specifies parameters that are valid in all other configuration
 contexts. They also serve as defaults for other configuration sections.
 
 ```yaml
 global:
-  # How frequently to scrape targets by default.
-  [ scrape_interval: <duration> | default = 1m ]
+# How frequently to scrape targets by default.
+[ scrape_interval: <duration> | default = 1m ]
+
+# How long until a scrape request times out.
+[ scrape_timeout: <duration> | default = 10s ]
 
-  # How long until a scrape request times out.
-  [ scrape_timeout: <duration> | default = 10s ]
+# How frequently to evaluate rules.
+[ evaluation_interval: <duration> | default = 1m ]
 
-  # How frequently to evaluate rules.
-  [ evaluation_interval: <duration> | default = 1m ]
+# The labels to add to any time series or alerts when communicating with
+# external systems (federation, remote storage, Alertmanager).
+external_labels:
+[ <labelname>: <labelvalue> ... ]
+```
 
-  # The labels to add to any time series or alerts when communicating with
-  # external systems (federation, remote storage, Alertmanager).
-  external_labels:
-    [ <labelname>: <labelvalue> ... ]
+### `<rule_files>`
 
+The list of files containing reporting and alerting rules.
+
+```yaml
 # Rule files specifies a list of globs. Rules and alerts are read from
 # all matching files.
 rule_files:
   [ - <filepath_glob> ... ]
-
-# A list of scrape configurations.
-scrape_configs:
```

Comment:
The top level config is missing now. That needs to stay in order to tie everything together.

Reply:
Added.

Reply:
I'm not seeing it, it should look fairly similar to the old one

Reply:
Can you clarify what you mean? I've broken out the exact file into three new sections, each with its own header.

Reply:
There's nothing tying them together, and telling me where scrape_configs goes and that it's at the same level as global.

Reply:
Okay - that's easy to fix.